eShopOnContainers/eShopOnContainers.Droid/Resources/xml/network_security_config.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+  <domain-config cleartextTrafficPermitted="true">
+    <!-- Debug port -->
+    <domain includeSubdomains="true">10.0.2.2</domain>
+    
```

Comment:
This is where you will add the endpoint for the backend for Android, iOS will just work